controllers/article-controllers/article-controllers.js

```
@@ -26,25 +28,54 @@ exports.getArticles = (req, res, next) => {
 exports.getCommentsByArticleID = (req, res, next) => {
   const { article_id } = req.params;
 
-    Promise.all([getArticleByID(article_id), getComments(article_id)])
+  Promise.all([getArticleByID(article_id), getComments(article_id)])
     .then(([article, comments]) => {
-        res.status(200).send({ article_comments: comments});
+      res.status(200).send({ article_comments: comments });
     })
     .catch((err) => {
-        if (err.status === 404) {
-            return next({ status:404, msg: "Can't find comments at provided article"})
-        }
-        next(err)
-    })
+      if (err.status === 404) {
+        return next({
```

could we make this use of the next middleware and not have to specify the error in this line? Maybe we throw the error in our model ?